### DISCUSSION TITLE
Chaos Recon database does not have http_urls for all domains available in public-bugbounty-programs

### DISCUSSION THREAD

**Discussion by tarunkant** [projectdiscovery/chaos-client#102] [General]
### chaos-client version:
0.2.0

### Current Behavior:
Chaos Recon database does not have http_urls for all domains available

### Expected Behavior:
It should have recon data for all domains available in public-bugbounty-programs

### Steps To Reproduce:
1. Run the following command:

```
chaos -d cloudamqp.com
```

Output:

```
advanced-meerkat-01.rmq.cloudamqp.com
alert-gazelle.rmq.cloudamqp.com
alpaca.cloudamqp.com
amqproxy.apt.cloudamqp.com
beige-barnacle-01.rmq.cloudamqp.com
blond-gopher-01.rmq.cloudamqp.com
blond-jellyfish.rmq.cloudamqp.com
blond-porpoise-01.rmq.cloudamqp.com
blue-dingo-01.rmq.cloudamqp.com
blue-dove-01.rmq.cloudamqp.com
brown-cockroach-01.rmq.cloudamqp.com
[...]
```

Command 2: 

```
chaos -d cloudamqp.com -bbq -http-url
```

Output:

```
No data
```

Manually running httpx on `chaos -d cloudamqp.com ` does show the data but Chaos recon data doesn't have any data for this.

**Comment by tarunkant** [comment]
Would this be a feature request @ehsandeep ?

**Comment by Mzack9999** [comment]
@tarunkant Results might not be aligned temporarily as chaos is under active development internally, soon all the data will be available again. I'm converting the issue to a discussion.

**Reply by MetzinAround** [reply]
@Mzack9999 Any update on this? Gonna close this discussion absent more info.
